### DISCUSSION TITLE
0.25.11 to PyPi?

### DISCUSSION THREAD

**Discussion by edwardreed81** [CamDavidsonPilon/lifelines#1257] [General]
I'm waiting on the Numpy version fix in 0.25.11 - it was released on this project a week ago, but hasn't hit PyPi yet, so I can't install it in the normal way with Poetry. Any idea when it'll be on there? 0.25.10 hit PyPi the same day it was released here, so not sure the process...

**Comment by CamDavidsonPilon** [comment]
Hey @edwardreed81 - oh weird, normally when I create a new [release](https://github.com/CamDavidsonPilon/lifelines/releases/tag/v0.25.11) in Github, a gh action will push to PyPI. I'll push 0.25.11 is in PyPI now, and investigate what went wrong.

**Reply by edwardreed81** [reply]
Cool, thanks!

**Reply by CamDavidsonPilon** [reply]
Okie it's up: https://pypi.org/project/lifelines/

Thanks for the heads up!
